packages/react-core/src/components/Masthead/examples/Masthead.md

```diff
@@ -0,0 +1,225 @@
+---
+id: Masthead
+section: components
+cssPrefix: pf-c-masthead
+propComponents: ['Masthead', 'MastheadToggle', 'MastheadMain', 'MastheadBrand', 'MastheadContent']
+---
+
+import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
+import { Link } from '@reach/router';
+
+## Examples
+
+### Basic
+
+```js
```

Comment:
Not sure how feasible it is, but you didn't just create a whole bunch of typescript examples in the demo app - should you be making these examples using typescript?

Reply:
I can swap the `js` to `ts`, though it won't change much for these examples. Is that all I have to do to get the docs to parse these as TypeScript?

Reply:
@redallen

Reply:
Yes that's correct @kmcfaul .